build.gradle

```diff
@@ -51,7 +51,11 @@ subprojects { subProject ->
   spotless {
     kotlin {
       target "**/*.kt"
-      ktlint(dep.ktlintVersion).userData(['indent_size': '2', 'continuation_indent_size': '4'])
+      ktlint(dep.ktlintVersion).userData([
```

Comment:
I'd actually remove spotless it seems to be indenting wrong..

Reply:
ok sure we can do that for now.